stubs/presets/laravel.stub

```diff
@@ -0,0 +1,2 @@
+csrf
+throwable
```

Comment:
@faissaloux Could you maybe move the `throwable` and `csrf` to the base stub? They are both common words in php web applications and not just laravel.